win32/src/threadpool_win32.c

```diff
@@ -337,9 +338,6 @@ int threadpool_schedule_work(THREADPOOL_HANDLE threadpool, THREADPOOL_WORK_FUNCT
                     /* Codes_SRS_THREADPOOL_WIN32_01_041: [ threadpool_schedule_work shall call SubmitThreadpoolWork to submit the work item for execution. ]*/
                     SubmitThreadpoolWork(ptp_work);
 
-                    (void)InterlockedDecrement(&threadpool->pending_api_calls);
-                    WakeByAddressSingle((PVOID)&threadpool->pending_api_calls);

```

Comment:
oopsie I suppose/

Reply:
No this got moved up to the work_function.